.github/workflows/ci.yml

```diff
@@ -15,14 +15,16 @@ jobs:
       - name: Setup node
         uses: actions/setup-node@v3
         with:
-          node-version: "16"
+          node-version: '16'
 
       - name: Install Dependencies
+        env:
+          MNEMONIC: 'fake fake fake fake fake fake fake fake fake fake fake fake fake fake fake fake fake fake fake fake fake fake fake fake'
```

Comment:
we should update the hardhat config to have a default if this is not set. I'll create an issue